### PR TITLE
Add support for aarch64

### PR DESCRIPTION
**Pull request by tyrsyna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 To build the image:
 
 ```
-./build.zsh node:14.15.4-alpine watchman-4.9.0-r2.apk
+./build.zsh node:14.15.4-alpine watchman-4.9.0-r2.apk x86_64
 ```
 
-The first argument is the docker image in which to build the apk and the second argument is the location to copy the apk to in the local filesystem. The arguments can be omitted in which case the defaults shown above are used.
+The first argument is the docker image in which to build the apk and the second argument is the location to copy the apk to in the local filesystem the third argument is for what architecture to build the apk (x86_64|aarch64). The arguments can be omitted in which case the defaults shown above are used.
```
Comment (insidewhy) [inline README.md]:
```suggestion
The first argument is the docker image in which to build the apk, the second argument is the location to copy the apk to in the local filesystem and the third argument specifies the architecture to use when building the apk (x86_64|aarch64). The arguments can be omitted in which case the defaults shown above are used.
```

**Comment by tyrsyna** [comment]
Changed.

**Review by penumbra1 — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (insidewhy) [inline .gitignore]: I think this is something better to put in `$HOME/.gitignore` rather than every repo.

Comment (insidewhy) [inline .gitignore]:
```suggestion
```

**Pull request merged**